docs/user-guide.md

```diff
@@ -5,6 +5,8 @@
 
 Asperathos is a platform to facilitate the deployment and control of applications running in cloud environments. For example, it can provide Quality of Service (QoS) by controlling resources allocated during runtime. Nevertheless, in contrast to other orchestration tools, such as Kubernetes itself or OpenStack Heat, it can be configured to consider application specific metrics and to actuate in a customized fashion. 
 
+![Asperathos Context](../imgs/context-asperathos.png)
```

Comment:
In the image, correct label: "submits and deletes jobs jobs" -> "submits and deletes jobs"

Reply:
Labels fixed. Image changed in commit 59cdc4d104b8a0e7ba3f89aab68eac8ee3d714b7